frontend/src/lib/utils.test.ts

```diff
@@ -34,6 +34,21 @@ describe('toParams', () => {
         const right = toParams({ a: 'b', ...{}, ...{}, b: 'c' })
         expect(left).toEqual(right)
     })
+
+    it('does not send empty string', () => {
```

Comment:
@mariusandra I needed to add/change the behaviour of the toParams method

Reply:
This might break things. We use this `toParams` in a lot of places when making API requests, and I can easily imagine some funnel thing breaking when the filter doesn't include an empty string, but is null/undefined instead.

Can you instead just send an `undefined` instead of `""` and it'll be cleaned up anyway?